Makefile

```diff
@@ -49,6 +49,15 @@ example-project:
 test:
 	go test -v --count=1 -p=1 ./...
 
+lint:
+	golangci-lint run ./...
+
+lint/fix:
+	golangci-lint run --fix ./...
+
+fix-imports:
+	gci write --Section Standard --Section Default --Section "Prefix(github.com/suborbital)" --Section "Prefix(github.com/suborbital/atmo)" ./**/*.go
```

Comment:
Doesn't `go fmt` handle this?

Reply:
this is currently unused until golangci-lint releases the next version after 1.44.0. The gci included in golangci-lint would flag the output of this as incorrect.

Essentially go fmt has some weirdness around sorting imports. Grouping is not great, blank identifier imports are weird, imports with comments are weird. I like using gci instead because it's always deterministic. See https://github.com/daixiang0/gci on what it does and what problems it solves.

I also like to separate my imports into 4 groups rather than three.

3 groups, default:

```
standard libs

3rd party libs

project libs
```
The shortcoming there is that when I'm working with `atmo`, I have a choice to make.
1. either I make `github.com/suborbital/atmo` the "project", in which case `github.com/suborbital/reactr` gets grouped into the 3rd party group, which I think is not the correct place for them to be, or
2. I make `github.com/suborbital` the "project", in which case both atmo and reactr get grouped into the third, local group project, which is also not correct.

with the new version of gci (0.3.0) we can create better groupings, so for atmo we can do
```
standard

3rd party

our org, not current project (reactr)

current project other modules (atmo)
```

This is something I would like to talk about in the how we work meeting next week. Going to look for a notion page for that, or create one if it doesn't exist.

Reply:
Added to https://www.notion.so/suborbital/d835656f929849fdb68550d0f58a54ea?v=b56bcf13260b49d2a8966d7500611df4&p=743e32ec8c84441db6bf2b301560ec53

Reply:
I'll remove this for now, pending our discussion on feb 24